core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -641,4 +641,27 @@ void defaultTcpUserTimeoutApplied() {
                 .containsExactly(entry(TCP_USER_TIMEOUT, userDefinedValue),
                                  entry(SO_LINGER, lingerMillis));
     }
+
+    @Test
+    void testServerLogUncaughtExceptions() {
```

Comment:
Could you extract this test into its own test class and add more, including:

- Make sure the message is actually logged when there were exceptions.
- Make sure the msssage is never logged when there were no excpetions.

Reply:
Sure I'll write those tests.